inst/extdata/tasks_demo/tasks_demo.R

```diff
@@ -0,0 +1,65 @@
+# functions run within the task plan
+process <- function(task, resource_C) {
+  if(isTRUE(getOption('scipiper.test_verbose'))) {
+    message(sprintf("   - processing %s, resource_C=%s", task, resource_C))
+  }
+  if(rnorm(1) < 0.1) stop("processing failed")
+  return(sprintf('PROCESSED_%s', task))
+}
+visualize <- function(prep, capital) {
+  if(isTRUE(getOption('scipiper.test_verbose'))) {
+    message(sprintf("   - visualizing with prep=%s, capital=%s", prep, capital))
+  }
+  if(rnorm(1) < 0.1) stop("visualizing failed")
```

Comment:
when you set a seed, does this succeed or fail every time in loop tasks? I was running into that on my example that I set up to test `progress`, but I didn't play around much w/ it after I rand into it. Perhaps it was where I was setting the seed in the code.

Reply:
and actually, I was using `sample()` instead of `rnorm()`...not sure if that matters

Reply:
I think both `sample` and `rnorm` use the same random number generator state determined by `set.seed`. As far as I could tell, setting the seed was working for making the failures/successes reproducible here. But I couldn't get R CMD check or Ctrl-Shift-T (devtools::test_somethingsomething) to work at all with this set of tests, for completely separate reasons, and I'm not sure whether Travis/linux would produce the same sequence of successes and failures.

Reply:
makes sense. Whatever was happening to me was unexpected, so I think there was something else going on.